packages/sdks/nextjs-sdk/README.md

```diff
@@ -0,0 +1,405 @@
+# Descope SDK for NextJS
+
+The Descope SDK for NextJS provides convenient access to the Descope for an application written on top of NextJS. You can read more on the [Descope Website](https://descope.com).
+
+This SDK uses under the hood the Descope React SDK and Descope Node SDK
+Refer to the [Descope React SDK](https://github.com/descope/descope-js/tree/main/packages/sdks/react-sdk) and [Descope Node SDK](https://github.com/descope/node-sdk) for more details.
+
+## Requirements
+
+- The SDK supports NextJS version 13 and above.
+- A Descope `Project ID` is required for using the SDK. Find it on the [project page in the Descope Console](https://app.descope.com/settings/project).
+
+## Installing the SDK
+
+Install the package with:
+
+```bash
+npm i --save @descope/nextjs-sdk
+```
+
+## Usage
+
+This section contains guides for App router and Pages router.
+For Pages router, see the [Pages Router](#pages-router) section.
+
+### App Router
+
+#### Wrap your app layout with Auth Provider
+
+```js
+// src/app/layout.tsx
+
+import { AuthProvider } from '@descope/nextjs-sdk';
+
+export default function RootLayout({
+	children
+}: {
+	children: React.ReactNode
+}) {
+	return (
+		<AuthProvider projectId="your-descope-project-id">
+			<html lang="en">
+				<body>{children}</body>
+			</html>
+		</AuthProvider>
+	);
+}
+```
+
+Note: `AuthProvider` uses `sessionTokenViaCookie` by default, in order that the [AuthMiddleware](<#Require-authentication-for-application-(Middleware)>) will work out of the box.
+
+#### Use Descope to render Flow
+
+You can use **default flows** or **provide flow id** directly to the Descope component
+
+```js
+// Login page, e.g. src/app/sign-in.tsx
+import { Descope } from '@descope/nextjs-sdk';
+// you can choose flow to run from the following without `flowId` instead
+// import { SignInFlow, SignUpFlow, SignUpOrInFlow } from '@descope/nextjs-sdk'
+
+const Page = () => {
+	return (
+		<Descope
+			flowId="sign-up-or-in"
+			onSuccess={(e) => console.log('Logged in!')}
+			onError={(e) => console.log('Could not logged in!')}
+			redirectAfterSuccess="/"
+			// redirectAfterError="/error-page"
+		/>
+	);
+};
+```
+
+Refer to the [Descope React SDK Section](https://github.com/descope/react-sdk?tab=readme-ov-file#2-provide-flow-id) for a list of available props.
```

Comment:
Need to fix url, please search fore descope/react-sdk in the entire repo

Reply:
fixed
couldn't find additional URLs